### ISSUE TITLE
Adapt the warning message on `CREATE EXTENSION`

### ISSUE THREAD

**Issue by bleskes** [duckdb/pg_duckdb#321]
### Description

Today we have:
```
WARNING:  To actually execute queries using DuckDB you need to run "SET duckdb.execution TO true;"
```

That's a bit scary and also is hard to parse for people who are new to our extension. I think it should say something like:
```
By default, DuckDB will be used to execute any query that requires data from blob storage or MotherDuck. To enable DuckDB's vectorized engine on all queries, run "SET duckdb.execution TO true;" 
```

Note that I removed the WARNING as well.

Thoughts?

**Comment by JelteF** [comment]
In #314 I removed the warning and updated the message in the README as well as changing the name to `duckdb.force_execution`. I think that solves this issues.

**Comment by bleskes** [comment]
Sounds great. Thanks!
